Code-the-Dream-School/bb-practicum-team-3-front
# Search Results

## Description
Added Search Results Page that will populate a list of hotels based on different criteria such as destination, checkin date, checkout date, number of guests, and number of rooms. 

*** Update: Merge and integrated search results page with the flow of the app. User should be able to start on the homepage and search a destination. Takes you to the search results page where you can see the hotels results and repeat the process in terms of searching. Researching on the search results  page just reloads the page with the new results. 




## Related Issue



## Acceptance Criteria
The page is responsive, console has no errors.


## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


<img width="1429" alt="image" src="https://user-images.githubusercontent.com/84487850/232175878-6c672da2-e61c-43a0-aa7a-98e36e43de93.png">

### After
![image](https://user-images.githubusercontent.com/74852114/232250844-d74a0514-4796-4dca-b01b-2c7aa18161d7.png)




## Testing Steps / QA Criteria
Start on the Homepage, search for a destination, review the list of hotels, check to see if hotel image, name and other details display properly, check for page responsiveness. When clicked on "See Availability" button, it should bring you to the Hotel Detail page.
